src/lib/storage/disk_size.rb

```diff
@@ -135,9 +172,9 @@ def unit_multiplier(unit)
       def +(other)
         return DiskSize.unlimited if any_operand_unlimited?(other)
         if other.is_a?(Numeric)
-          DiskSize.new(@size_k + other)
-        elsif other.respond_to?(:size_k)
-          DiskSize.new(@size_k + other.size_k)
+          DiskSize.new(@size_b + other)
```

Comment:
This may look NP, but you use `@size_b` directly everywhere in the file. I you would use `size_b` (the accessor instead of directly the instance variable), it would be more robust to changes in the future. Hopefully we will not change the internal unit again but in general I like to avoid direct access to instance variables, even within the class (uniform access principle).